test/functional/http/get/transactions.js

```diff
@@ -383,6 +398,52 @@ describe('GET /api/transactions', () => {
 			});
 		});
 
+		describe('senderIdOrRecipientId', () => {
+			it('using invalid senderIdOrRecipientId should fail', () => {
+				return transactionsEndpoint
+					.makeRequest({ senderIdOrRecipientId: '' }, 400)
+					.then(res => {
+						expectSwaggerParamError(res, 'senderIdOrRecipientId');
+					});
+			});
+
+			it('using one senderIdOrRecipientId should return incoming and outgoing transaction of an account', () => {
+				var sender = [];
+				var recipient = [];
+				var accountId = account.address;
+				return transactionsEndpoint
+					.makeRequest({ senderIdOrRecipientId: accountId }, 200)
+					.then(res => {
+						expect(res.body.data).to.not.empty;
+						res.body.data.map(transaction => {
+							if (accountId === transaction.recipientId) {
+								recipient.push(transaction);
+							} else if (accountId === transaction.senderId) {
+								sender.push(transaction);
+							}
+						});
+						expect(sender).to.have.length(1);
+						expect(recipient).to.have.length(1);
+					});
+			});
+
+			it('using multiple senderIdOrRecipientId should fail', () => {
+				return transactionsEndpoint
+					.makeRequest(
+						{
+							senderIdOrRecipientId: [
+								accountFixtures.genesis.address,
+								accountFixtures.existingDelegate.address,
+							],
+						},
+						400
+					)
+					.then(res => {
+						expectSwaggerParamError(res, 'senderIdOrRecipientId');
+					});
+			});
```

Comment:
This definitely will fail, as the format of the param you specified in swagger is `string` not the array. So I don't think its a right test to be added.